conda_smithy/templates/drone.yml.tmpl

```diff
@@ -0,0 +1,27 @@
+{% if configs[0] -%}
+{%- for config, platform, upload in configs | sort -%}
```

Comment:
Instead of repeating the same configuration for multiple times, can Matrix builds be used?

Reply:
Unfortunately matrix support was removed in 1.0.  Each pipeline runs in parallel but the config mostly gets duplicated.

Reply:
Is there support for yaml anchors?

Reply:
I don't think so but I will double check.  I think the approach Drone uses for managing complicated configurations is to leverage Jsonnet to generate them.

Reply:
Okay, if yaml anchors are not supported, then this is fine

Reply:
Sorry I misspoke - it does look like anchors are supported.  Apparently Drone uses the `go-yaml` library, which (I was told) should work, but can have some issues.

Reply:
@isuruf I think I have the anchors worked out.

Reply:
Thanks. Can you send a WIP PR to numpy-feedstock after rendering using this branch?

Reply:
Done.